Backend/UserService.cs

```diff
@@ -119,5 +119,32 @@ public async Task<IdentityResult> UpdateUserAsync(User userToUpdate)
         {
             return await _userManager.UpdateAsync(userToUpdate);
         }
+
+        public UserInformations GetUserFromLDAP(string login, string password)
+        {
+            login += "@fri.uniza.sk";
```

Comment:
What about if user already write the email also?

Reply:
Will return error that should just put student name. It is in cotroller. Its because normally students email is @stud.uniza.sk.

Reply:
OK